ProjectHive/ProjectHive.Services.NotificationsAPI/Services/WebSocketHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json;
+using ProjectHive.Services.NotificationsAPI.Dto;
+using ProjectHive.Services.NotificationsAPI.Models;
+using System.Net.WebSockets;
+using System.Text;
+
+namespace ProjectHive.Services.NotificationsAPI.Services
+{
+    public class WebSocketHandler :IWebSocketHandler
+    {
+        private readonly List<WebSocket> _sockets = new List<WebSocket>();
+
```

Comment:
Please do some tests, is this implementation thread safety? if not needs to add some thread-safe mechanisms